### PR TITLE
fix docs: changed link to virtualenv website

### PR DESCRIPTION
**Pull request by vlt**
Hello.

As freenode user lantern pointed out the docs point to a parked page notice for virtualenv. I fixed that.

### CODE REVIEW

File: docs/howto/upgrade-version.txt [hunk 1]
```diff
@@ -95,7 +95,7 @@ If you use some other installation process, you might have to manually
 should look at the complete installation instructions.
 
 .. _pip: https://pip.pypa.io/
-.. _virtualenv: http://www.virtualenv.org/
+.. _virtualenv: https://virtualenv.pypa.io//
```
Comment (timgraham) [inline docs/howto/upgrade-version.txt]: double slash typo

Comment (vlt) [inline docs/howto/upgrade-version.txt]: Oh, thanks! Fixed that :-)

**Comment by vlt** [comment]
Today it redirects for me as well.

I checked the whois data:

    Last update of WHOIS database: 2016-11-22T08:04:41Z
    Updated Date: 2016-11-20T02:53:07Z
    Registrant Organization: Python Software Foundation

Yes, there has been a recent change. Hmmm …

**Comment by timgraham** [comment]
merged in 66505b72c58099ded7420a7f90792d5bdf48ff01, thanks.

**Pull request closed without merging**